recursive-list-main/src/components/breadcrumbs/breadcrumbs.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+import { Breadcrumbs } from "components";
+import { MemoryRouter, Route, Routes } from "react-router-dom";
+
+describe("Breadcrumbs", () => {
+  it("display home breadcrumb by default", () => {
+    render(
+      <MemoryRouter>
+        <Breadcrumbs />
+      </MemoryRouter>
+    );
+
+    expect(screen.getByTestId("home_svg")).toHaveAttribute("href", "/");
```

Comment:
check naming conventions that can be used for test if, i prefer kebab case (home-svg)

Reply:
Modified file name with kebab case